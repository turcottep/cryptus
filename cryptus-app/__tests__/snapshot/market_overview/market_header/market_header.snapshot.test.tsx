import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";

import MarketHeader from "../../../../components/market_overview/market_header/market_header";

describe("<MarketHeader />", () => {
  it("should display properly", async () => {
    const tree = renderer.create(<MarketHeader />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
