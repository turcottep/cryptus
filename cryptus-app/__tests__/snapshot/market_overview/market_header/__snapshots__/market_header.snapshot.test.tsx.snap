// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MarketHeader /> should display properly 1`] = `
<div>
  <div>
    March 12
  </div>
</div>
`;
